addons/snailmail/static/src/scss/thread.scss

```diff
@@ -0,0 +1,17 @@
+.o_thread_message_snailmail {
+    padding: 4px;
+
+    &.o_thread_message_snailmail_canceled, &.o_thread_message_snailmail_pending, &.o_thread_message_snailmail_sent {
```

Comment:
with newly introduced classname for style of error, you can use `:not(.<classname>)`